chatops4s-discord-example/src/main/scala/services/SendToProductionService.scala

```diff
@@ -0,0 +1,30 @@
+package services
+
+import cats.effect.IO
+import com.typesafe.scalalogging.{Logger, StrictLogging}
+
+class SendToProductionService extends StrictLogging {
+  private val discordOutbound = new DiscordOutbound()
```

Comment:
this you would usually pass through constructor, no need to recreate everywhere its needed.